src/ws/cockpit-desktop.in

```diff
@@ -53,7 +53,11 @@ class Browser(Gtk.Window):
 
         self.set_size_request(800, 600)
 
-        WebKit2.WebContext.get_default().set_sandbox_enabled(True)
+        context = WebKit2.WebContext.get_default()
+        data_manager = context.get_website_data_manager()
+        data_manager.set_network_proxy_settings(WebKit2.NetworkProxyMode.NO_PROXY, None)
+        context.set_sandbox_enabled(True)
```

Comment:
That's obviously fine. Just curious, how did that break flatpak? Or how did you notice?

Reply:
The flatpak is fairly strictly constrained, and the "lookup the system proxy settings" required a call to a DBus interface that I wasn't allowed to access.  That broke with a vague error message, and when I enabled debugging output I noticed it was trying to get the proxy settings.  Hardwiring this to "off" fixed it.